resource-timing/resource_initiator_types.html

```diff
@@ -64,40 +65,88 @@
-    expected_entries[pathname + 'blank_page_green.htm'] = 'iframe';
-    expected_entries[pathname + 'empty_script.js'] = 'script';
-    expected_entries[pathname + 'resource_timing_test0.css?id=embed'] = 'embed';
-    expected_entries[pathname + 'resource_timing_test0.css?id=n1'] = 'css';
-    expected_entries[font_pathname + '?id=n1'] = 'css';
-    expected_entries[pathname + 'blue.png?id=1'] = 'css';
-    expected_entries[pathname + 'blue.png?id=2'] = 'css';
-    expected_entries[pathname + 'blue.png?id=async_xhr'] = 'xmlhttprequest';
-    expected_entries[pathname + 'blue.png?id=body'] = 'body';
-    expected_entries[pathname + 'blue.png?id=input'] = 'input';
-    expected_entries[pathname + 'blue.png?id=n1'] = 'css';
-    expected_entries[pathname + 'blue.png?id=object'] = 'object';
-    expected_entries[pathname + 'blue.png?id=poster'] = 'video';
-    expected_entries[pathname + 'nested.css'] = 'link';
+    addEntryIfExists(entries, expected_entries, font_pathname, 'css');
+    addEntryIfExists(entries, expected_entries, pathname + 'resource_timing_test0.png', 'img');
+    addEntryIfExists(entries, expected_entries, pathname + 'resource_timing_test0.png?id=srcset-srcset', 'img');
+    addEntryIfExists(entries, expected_entries, pathname + 'resource_timing_test0.png?id=srcset-src', 'img');
+    addEntryIfExists(entries, expected_entries, pathname + 'blank_page_green.htm', 'iframe');
+    addEntryIfExists(entries, expected_entries, pathname + 'blank_page_green.htm?id=frame', 'frame');
+    addEntryIfExists(entries, expected_entries, pathname + 'empty_script.js', 'script');
+    addEntryIfExists(entries, expected_entries, pathname + 'resource_timing_test0.css?id=embed', 'embed');
+    addEntryIfExists(entries, expected_entries, pathname + 'resource_timing_test0.css?id=n1', 'css');
+    addEntryIfExists(entries, expected_entries, font_pathname + '?id=n1', 'css');
+    addEntryIfExists(entries, expected_entries, pathname + 'blue.png?id=cursor', 'css');
+    addEntryIfExists(entries, expected_entries, pathname + 'blue.png?id=1', 'css');
+    addEntryIfExists(entries, expected_entries, pathname + 'blue.png?id=2', 'css');
+    addEntryIfExists(entries, expected_entries, pathname + 'blue.png?id=async_xhr', 'xmlhttprequest');
+    addEntryIfExists(entries, expected_entries, pathname + 'blue.png?id=body', 'body');
+    addEntryIfExists(entries, expected_entries, pathname + 'blue.png?id=input', 'input');
+    addEntryIfExists(entries, expected_entries, pathname + 'blue.png?id=n1', 'css');
+    addEntryIfExists(entries, expected_entries, pathname + 'blue.png?id=object', 'object');
+    addEntryIfExists(entries, expected_entries, pathname + 'blue.png?id=video-poster', 'video');
+    addEntryIfExists(entries, expected_entries, '/media/test.mp4?id=video-src', 'video');
+    addEntryIfExists(entries, expected_entries, '/media/test.mp4?id=video-source', 'source');
+    addEntryIfExists(entries, expected_entries, '/media/test.ogg?id=video-source', 'source');
```

Comment:
@nicjansma did you mean test.ogv? There is no test.ogg under media.

Reply:
Sent https://github.com/w3c/web-platform-tests/pull/10077 to you. PTAL.

Reply:
Yes!  Thank you for finding this